Sources/SpeziValidation/ValidationState/ValidationState.swift

```diff
@@ -67,7 +67,7 @@ public struct ValidationState: DynamicProperty {
 extension ValidationState {
     /// A binding to a ``ValidationState``.
     @propertyWrapper
-    public struct Binding {
+    public struct Binding: Sendable {
```

Comment:
This `Sendable` annotaion is easily possible as `ValidationContext` is now `Sendable` and a SwiftUI `Binding` is `Sendable` as long as the wrapped value is `Sendable` 👍